examples/minimal/snowpack.config.js

```diff
@@ -0,0 +1,3 @@
+export default {
+	plugins: [],
+}
```

Comment:
This file can be removed! Snowpack defaults `plugins` to `[]`

Reply:
Thank you. I’ll remove this file!

Reply:
This has been removed in dfc2525